orchestration/dagster_orchestration/hca_orchestration/pipelines/__init__.py

```diff
@@ -0,0 +1,8 @@
+from hca_orchestration.pipelines.stage_data import stage_data
+from hca_orchestration.pipelines.validate_egress import validate_egress
+
+
+__all__ = [
+    stage_data,
+    validate_egress,
+]
```

Comment:
not sure if this needs an explanatory comment - this controls what happens when you call `from hca_orchestration.pipelines import *`

generally this file lets us import the pipelines themselves directly from `hca_orchestration.pipelines` instead of needing to go into their individual implementation files.